### ISSUE TITLE
Cant understand these errors please help! 

### ISSUE THREAD

**Issue by akashrpatil** [christophetd/censys-subdomain-finder#10]
root@kali: python3 censys_subdomain_finder.py -h
censys_subdomain_finder.py:41: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if len(subdomains) is 0:
censys_subdomain_finder.py:53: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if output_file is None or len(subdomains) is 0:

root@kali: python censys_subdomain_finder.py -h
Traceback (most recent call last):
  File "censys_subdomain_finder.py", line 3, in <module>
    import censys.certificates
ImportError: No module named censys.certificates

Note: i tried with python2 and python3 and also install all the reqirements using pip3 install -r requirement.txt

**Comment by penetrationexpert** [comment]
hello thanx for response i get result of -h but with these errors:

(venv) root@kali:~/tool/censys-subdomain-finder# python censys_subdomain_finder.py -h
```censys_subdomain_finder.py:41: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if len(subdomains) is 0:
censys_subdomain_finder.py:53: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if output_file is None or len(subdomains) is 0:```
usage: censys_subdomain_finder.py [-h] [-o OUTPUT_FILE] [--censys-api-id CENSYS_API_ID] [--censys-api-secret CENSYS_API_SECRET]
                                  domain

positional arguments:
  domain                The domain to scan

optional arguments:
  -h, --help            show this help message and exit
  -o OUTPUT_FILE, --output OUTPUT_FILE
                        A file to output the list of subdomains to (default: None)
  --censys-api-id CENSYS_API_ID
                        Censys API ID. Can also be defined using the CENSYS_API_ID environment variable (default: None)
  --censys-api-secret CENSYS_API_SECRET
                        Censys API secret. Can also be defined using the CENSYS_API_SECRET environment variable (default: None)

**Comment by christophetd** [comment]
fixed in #14
